src/services/training/index.html

```diff
@@ -33,6 +32,20 @@
           %}
       </div>
   </section>
+  <section class="training-overview-katalyst-banner">
+    {% capture title %}{% t  services.training.katalyst-banner.title %}{% endcapture %}
+    {% capture description %}{% t  services.training.katalyst-banner.description %}{% endcapture %}
+    {% capture hubspot-id %}{% t  services.training.katalyst-banner.hubspot-form-id %}{% endcapture %}
+    {% capture hubspot-cta %}{% include hubspot_cta.liquid class="hubspot-in-page-banner" hubspot-id=hubspot-id %}{% endcapture %}
+    {% include in_page_banner.html 
+      title=title
+      description=description
+      class="training-overview__katalyst-banner" 
+      background_image="/assets/custom/img/training/training-overview-katalyst-banner.png"
+      background_position="30% center"
+      hubspot-cta=hubspot-cta
```

Comment:
Not a biggie but I find it pleasant to alphabetically sort an attribute lists that need to be broken over multiple lines. It makes it easier to find things, easier to spot duplicates and less chance of merge conflicts if two people change the same list. 😊